### PR TITLE
(D36681) FreeBSD Handbook: FreeBSD Boot Process: directions for users of GNU GRUB

### PR DESCRIPTION
**Pull request by grahamperrin**
Arising from <https://old.reddit.com/r/freebsd/comments/xhkym9/-/> about GRUB. 

Add a link to FreeBSD FAQ, where there's a brief answer; and to GNU-provided documentation. 

GRUB should be uppercase.

### CODE REVIEW

File: documentation/content/en/books/handbook/boot/_index.adoc [hunk 1]
```diff
@@ -103,7 +103,12 @@ Work is under way to provide direct UEFI booting.
 
 The code within the MBR is typically referred to as a _boot manager_, especially when it interacts with the user.
 The boot manager usually has more code in the first track of the disk or within the file system.
-Examples of boot managers include the standard FreeBSD boot manager boot0, also called Boot Easy, and Grub, which is used by many Linux(R) distributions.
+Examples of boot managers include the standard FreeBSD boot manager boot0, also called Boot Easy, and GNU GRUB, which is used by many Linux(R) distributions.
+
+[NOTE]
+====
+There's :faq:[a frequently asked question] about GRUB. Beyond the answer there, users of GRUB should please refer to https://www.gnu.org/software/grub/grub-documentation.html[GNU-provided documentation].
```
Comment (grahamperrin) [inline documentation/content/en/books/handbook/boot/_index.adoc:110]: Markup here is wrong. 

From <https://docs.asciidoctor.org/asciidoc/latest/syntax-quick-reference/#links> for _links_, I can't tell how to fix the link to <https://docs.freebsd.org/en/books/faq/#grub-loader>.

Comment (pauamma) [reply documentation/content/en/books/handbook/boot/_index.adoc:110]: See https://docs.freebsd.org/en/books/fdp-primer/asciidoctor-primer/#asciidoctor-links and specifically https://docs.freebsd.org/en/books/fdp-primer/asciidoctor-primer/#asciidoctor-links-internal .

**Comment by grahamperrin** [comment]
<https://reviews.freebsd.org/D36681>

**Comment by grahamperrin** [comment]
### Pushed to `freebsd HEAD:main`

* <https://reviews.freebsd.org/R9:b73963c22db3>
* <https://cgit.freebsd.org/doc/commit/?id=b73963c22db3>
* <https://github.com/freebsd/freebsd-doc/commit/b73963c22db3>
* <https://gitlab.com/FreeBSD/freebsd-doc/-/commit/b73963c22db3>

**Comment by grahamperrin** [comment]
Now under <https://docs.freebsd.org/en/books/handbook/boot/#boot-introduction>: 

image

**Comment by grahamperrin** [comment]
### Notes to self

1. git -C /usr/doc config --list | sort
2. git -C /usr/doc fetch origin patch-3
3. git -C /usr/doc switch patch-3
4. git -C /usr/doc pull
5. git -C /usr/doc diff -U999999 main > /tmp/D36681.diff
6. less /tmp/D36681.diff
7. git -C /usr/doc switch main
8. gh repo sync grahamperrin/freebsd-doc
9. git -C /usr/doc apply /tmp/D36681.diff
10. nano /tmp/D36681.diff
11. git -C /usr/doc stash
12. git -C /usr/doc stash clear
13. git -C /usr/doc apply /tmp/D36681.diff
14. git -C /usr/doc status
15. git -C /usr/doc add . && git -C /usr/doc commit -a
16. git -C /usr/doc push freebsd HEAD:main
17. git -C /usr/doc branch -D patch-3

Partial transcript – there was a trailing whitespace error: 

```text
…
% git -C /usr/doc apply /tmp/D36681.diff
/tmp/D36681.diff:117: trailing whitespace.
Users of GRUB should refer to https://www.gnu.org/software/grub/grub-documentation.html[GNU-provided documentation]. 
warning: 1 line adds whitespace errors.
% nano /tmp/D36681.diff
…
% git -C /usr/doc status
On branch main
Your branch is up to date with 'origin/main'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
        modified:   documentation/content/en/books/handbook/boot/_index.adoc

no changes added to commit (use "git add" and/or "git commit -a")
…
```

**Comment by bsdimp** [comment]
Git arc makes a lot of this easier...

**Comment by grahamperrin** [comment]
> Git arc makes a lot of this easier...

Thanks, I looked at and around [Create a Revision via command line (if using git)](https://wiki.freebsd.org/Phabricator#Create_a_Revision_via_command_line_.28if_using_git.29) around four months ago, a few things drove me in a different direction (away from Arcanist). 

I might revisit @bsdjhb's [FreeBSD Code Review with git-arc](https://issue.freebsdfoundation.org/publication/?m=33057&i=726773&p=8&ver=html5) | [PDF](https://freebsdfoundation.org/wp-content/uploads/2021/11/FreeBSD-Code-Review-with-git-arc.pdf) and have a chat in `#devguide` in Discord … some time in October, maybe.

**Comment by bsdimp** [comment]
That's a good article. Git ar. Has improved a bit since then. I never use 'arc' directly since it
 gets so many things wrong...

I'd also just use git checkout branch to switch branches... 

I also never work in /usr/{src,doc,ports} directly... saves the -C args as well for a simpler experience.

**Comment by pauamma** [comment]
> Git arc makes a lot of this easier...

1. /me makes a note to mention freebsd-git-devtools in the Committer's Guide, probably section 8.

**Pull request closed without merging**